### DISCUSSION TITLE
Study Aid Application

### DISCUSSION THREAD

**Discussion by alfmelchor** [TomSchimansky/CustomTkinter#1507] [Showcase]
I've started working on an application called Study Aid. This application allows users to create assignments and keep track of their statuses. Additionally, users can time themselves and see how far they study or set a timer to improve productivity. I've barely pushed it to GitHub and many changes are to come! 
https://github.com/alfmelchor/StudyAid

**Comment by avalon60** [comment]
Nice! Where did you get the icons from?

**Reply by alfmelchor** [reply]
They were made in Adobe Illustrator by me but were referenced from free-to-use websites :)!
